gettsim/social_insurance.py

```diff
@@ -26,185 +24,193 @@ def soc_ins_contrib(person, tb):
     # a couple of definitions
 
     # As there is only one household, we selcet west_ost dependent paramter in the
-    # beginning and place them in a seperate dictionary tb_ost.
+    # beginning and place them in a seperate dictionary params_ost.
     westost = "o" if person["east"] else "w"
-    tb_ost = {}
+    params_ost = {}
     for val in ["bezgr_", "mini_grenze", "kvmaxek", "rvmaxek"]:
-        tb_ost[val] = tb[val + westost]
+        params_ost[val] = params[val + westost]
 
-    # ssc["above_thresh_kv"] = inout["m_wage"] > tb_ost["kvmaxek"]
+    # ssc["above_thresh_kv"] = inout["m_wage"] > params_ost["kvmaxek"]
     #
-    # ssc["above_thresh_rv"] = inout["m_wage"] > tb_ost["rvmaxek"]
+    # ssc["above_thresh_rv"] = inout["m_wage"] > params_ost["rvmaxek"]
 
     # This is probably the point where Entgeltpunkte should be updated as well.
 
     # Check if wage is below the mini job grenze.
-    belowmini = person["m_wage"] < tb_ost["mini_grenze"]
+    belowmini = person["m_wage"] < params_ost["mini_grenze"]
 
     # Check if wage is in Gleitzone / Midi-Jobs
-    in_gleitzone = (tb["midi_grenze"] >= person["m_wage"]) & (
-        person["m_wage"] >= tb_ost["mini_grenze"]
+    in_gleitzone = (params["midi_grenze"] >= person["m_wage"]) & (
+        person["m_wage"] >= params_ost["mini_grenze"]
     )
 
     # Calculate accordingly the ssc
     if belowmini:
         person[OUT_COLS] = 0.0
     elif in_gleitzone:
-        # TODO: Before and in 2003 tb["midi_grenze"] is 0 and therefore we won't reach
-        #  this.
-        person = tb["calc_midi_contrib"](person, tb)
+        # TODO: Before and in 2003 params["midi_grenze"] is 0 and
+        #  therefore we won't reach this.
+        person = params["calc_midi_contrib"](person, params)
```

Comment:
Do we store a function in `params` or how does this construct work?

Reply:
Yes. In Juli when we sat together with Eric, you said that we should get this year checks out of the function and assign externally functions to the dictionary depending on the year!

Reply:
Haha, that's why we need those GEPs...

Reply:
So true!!!!